### PR TITLE
UI: Fix editableList inherits background-color of the group

### LINKED ISSUE #8302: Editable list property type has incorrect background color when placed inside property group
### Operating System Info

Ubuntu 20.04

### Other OS

_No response_

### OBS Studio Version

29.0.1

### OBS Studio Version (Other)

_No response_

### OBS Studio Log URL

n/a

### OBS Studio Crash Log URL

_No response_

### Expected Behavior

I would expect that the value container has a background color matching that of the other value containers (see screenshot)

### Current Behavior

The value container inherits the background color of the group it's in:

image

### Steps to Reproduce

1. Create a source that displays an editable text property inside of a property group
2. Open the properties dialog for that source

### Anything else we should know?

_No response_

### PR DESCRIPTION
**Pull request by tuduweb**
### Description

In the Yami theme, when a ediableList (`QlistWidget`) inside a `QGroupBox`, the background color are same, not easy to distinguish.

So change the `QListWidget` background color to other (*the comboBox background color*).

after:
Yami
Screenshot 2023-02-20 111930

light:
light-Screenshot 2023-02-21 003335

grey:
Grey-Screenshot 2023-02-21 003300

acri:
Acri-Screenshot 2023-02-21 003223

### Motivation and Context

fixes #8302 

the background-color of `QListWidget`(`QAbstractItemView`) same to `QGroupBox` and `PropertiesContainer`

### How Has This Been Tested?

Win10
Just rebuild and look at the background changes.

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by Warchamp7** [comment]
I don't like using the input color for the list here, but alas the Yami palette wasn't designed with 4 levels of hierarchy in mind so this will have to do for now.

File: UI/data/themes/Yami.qss [hunk 1]
```diff
@@ -273,6 +273,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
+#PropertiesContainer QListWidget {
+    background-color: rgb(60,64,75);
+}
+
```
Comment (Warchamp7) [inline UI/data/themes/Yami.qss:280]: Please use `palette(button)` here instead of the rgb value

File: UI/data/themes/Light.qss [hunk 2]
```diff
@@ -269,6 +269,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
+#PropertiesContainer QListWidget {
+    background-color: rgb(243,243,243);
```
Comment (Warchamp7) [inline UI/data/themes/Light.qss:274]: Please use `palette(button)` here instead of the rgb value

File: UI/data/themes/Grey.qss [hunk 3]
```diff
@@ -269,6 +269,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
+#PropertiesContainer QListWidget {
+    background-color: rgb(67,67,67);
```
Comment (Warchamp7) [inline UI/data/themes/Grey.qss:274]: Please use `palette(button)` here instead of the rgb value

File: UI/data/themes/Acri.qss [hunk 4]
```diff
@@ -269,6 +269,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
```
Comment (Warchamp7) [inline UI/data/themes/Acri.qss:272]: This sort of comment is not needed in qss files, especially as this is a flaw of the theme and not a bug per se

File: UI/data/themes/Grey.qss [hunk 5]
```diff
@@ -269,6 +269,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
```
Comment (Warchamp7) [inline UI/data/themes/Grey.qss:272]: Please remove

File: UI/data/themes/Light.qss [hunk 6]
```diff
@@ -269,6 +269,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
```
Comment (Warchamp7) [inline UI/data/themes/Light.qss:272]: Please remove

File: UI/data/themes/Yami.qss [hunk 7]
```diff
@@ -273,6 +273,11 @@ OBSBasicSettings #PropertiesContainer {
     background-color: palette(dark);
 }
 
+/* EdiableList inside Group Fix, issues#8302 */
```
Comment (Warchamp7) [inline UI/data/themes/Yami.qss:276]: Please remove

**Review by Warchamp7 — CHANGES_REQUESTED** [review]
I left some notes for fixes.

The commit and PR title also need the typo corrected, or adjusted to something more appropriate such as "Fix QListWidget background color in property groups"

File: UI/data/themes/Acri.qss (hunk 4 above)
Comment (tuduweb) [reply UI/data/themes/Acri.qss]: > This sort of comment is not needed in qss files, especially as this is a flaw of the theme and not a bug per se

thx, I will pay attention next time

**Comment by tuduweb** [comment]
Thx for your review, I have updated this PR.

**Review by Warchamp7 — APPROVED** [review]

**Review by RytoEX — APPROVED** [review]
> @RytoEX this was approved by @Warchamp7, this could potentially also fix a related issue with the contributor box in the "About" window (background colour is identical to scrollbar colour until the widget is selected). I guess this is good to merge?

@PatTheMav Sure, seems fine.

**Pull request merged**